feature/platform/tests/optics_power_and_bias_current_test/optics_power_and_bias_current_test.go

```diff
@@ -179,28 +166,22 @@ func TestOpticsPowerUpdate(t *testing.T) {
 	}
 }
 
-// findTransceiverName provides name of transciever port corresponding to interface name
+// findTransceiverName provides name of transceiver port corresponding to interface name
 func findTransceiverName(dut *ondatra.DUTDevice, interfaceName string) (string, error) {
-	var (
-		transceiverMap = map[ondatra.Vendor]string{
-			ondatra.ARISTA:  " transceiver",
-			ondatra.CISCO:   "",
-			ondatra.JUNIPER: "",
-			ondatra.NOKIA:   "-transceiver",
-		}
-	)
-	transceiverName := interfaceName
-	name, ok := transceiverMap[dut.Vendor()]
-	if !ok {
-		return "", fmt.Errorf("No transceiver interface available for DUT vendor %v", dut.Vendor())
-	}
-	if name != "" {
+	switch dut.Vendor() {
+	case ondatra.ARISTA:
+		suffix := " transceiver"
 		interfaceSplit := strings.Split(interfaceName, "/")
 		interfaceSplitres := interfaceSplit[:len(interfaceSplit)-1]
-		transceiverName = strings.Join(interfaceSplitres, "/") + name
-		if dut.Vendor() == ondatra.NOKIA {
-			transceiverName = interfaceName + name
-		}
+		return strings.Join(interfaceSplitres, "/") + suffix, nil
+	case ondatra.CISCO:
+		return "", nil
+	case ondatra.JUNIPER:
+		return "", nil
+	case ondatra.NOKIA:
+		suffix := "-transceiver"
+		interfaceName = strings.Replace(interfaceName, "ethernet", "Ethernet", 1)
+		return interfaceName + suffix, nil
 	}
-	return transceiverName, nil
+	return "", fmt.Errorf("no transceiver interface available for DUT vendor %v", dut.Vendor())
```

Comment:
To make a consistent `switch` syntax, let's move this to the `default` case. Also, I think we can remove the Cisco and Juniper case, and just make `default` to return the `interfaceName` (no error on not specified vendor).